### DISCUSSION TITLE
Suricata Alerts Stop Generating

### DISCUSSION THREAD

**Discussion by Cra5hedC0w** [Security-Onion-Solutions/securityonion#3135] [General] [answered]
I have an issue where SO stops generating suricata alerts. It's been at least  a few days since suricata alerts were generated in the SOC screen. The so-status shows everything running, and okay. I did a tcpdump and see traffic on the monitor port as well. I know for a fact the network traffic should be firing off alerts. 
I am receiving wazuh alerts however. This is a standalone install.  I am on the latest release.

Any ideas on where to begin with this?

**Comment by Cra5hedC0w** [comment, marked as answer]
This issue is resolved.

**Reply by ErwinDFIRocks** [reply]
Hi what was the issue and how did you solve this?

Thanks and Regards!

**Reply by Cra5hedC0w** [reply]
This issue was related to a switch configuration unrelated to Security Onion. The ports that were mirroring traffic stopped mirroring when the configuration was wiped on the switch after a power failure.
